bin/postinstall

```diff
@@ -139,6 +139,22 @@ setup_subversion_wrapper() {
 
 
 ##
+# Create a vhost for the repository wrapper to be accessed by localhost
+setup_subversion_vhost() {
+	local src="conf/vhost/repoman_svn_vhost.conf"
```

Comment:
Could you create a new `conf/server` directory instead, so that it's easier to see at which level the conf files get included?